tests/Unit/Document/Glossary/GlossaryTest.php

```diff
@@ -31,6 +33,9 @@ class GlossaryTest extends TestCase
      */
     protected function setUp(): void
     {
+        if(!GlossaryBundle::isInstalled()) {
```

Comment:
Glossary tests are [skipped](https://github.com/pimcore/pimcore/actions/runs/3730712672/jobs/6328082354#step:11:3218)at the moment. The bundle needs to be installed before running tests like we do it for ecommerce bundle here https://github.com/pimcore/pimcore/actions/runs/3730712672/jobs/6328082354#step:11:3218

Reply:
Is it enough to write a Helper like this one?
https://github.com/pimcore/pimcore/blob/11.x/tests/Support/Helper/Ecommerce.php

Reply:
we already have a helper for Unit suite https://github.com/pimcore/pimcore/blob/56d742deb8e61b33d34040714458460fa774f7fb/tests/Support/Helper/Unit.php#L23

Reply:
maybe we don't need the skip check anymore since the bundle is always installed.